### PR TITLE
Add pre commit hook

### LINKED ISSUE #59: Add husky
Add husky and run eslint & prettier pre commit

### PR DESCRIPTION
**Pull request by augusto-jm-amaral**
Using husky I add pre-commit hook running eslint and prettier check.

closes #59

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -94,6 +94,7 @@ export const getAllData = async (list, all = false) => {
     data.states = statesData;
   }
 
+<<<<<<< HEAD
```
Comment (mahowa) [inline src/data.js]: Not sure what happened here but if you could fix the merge conflict text that would be great

**Review by mahowa — CHANGES_REQUESTED** [review]
Just need to fix the conflicted file (Even tho git is saying its not)

File: src/data.js (hunk 1 above)
Comment (augusto-jm-amaral) [reply src/data.js]: Oh sorry. @mahowa, I will fix it

**Comment by augusto-jm-amaral** [comment]
@mahowa I fix the conflict, the saw too that the tests are break because this commit https://github.com/mahowa/Covid-19-d3/commit/87052dbdeb3c6a8c79a9758099fe9efc7e6455bb, would you like that I fix this?

What do you think if we check the tests/lint/prettier using GitHub actions? It is free and pretty cool :)

**Comment by mahowa** [comment]
Yeah fixing that test would be great!. I think thats a great Idea Ill see if I can get it set up

**Comment by augusto-jm-amaral** [comment]
I fix the tests @mahowa 👍

If you want help, this is an example of a workflow that I use in another project:
```
name: Lint and Tests

on: [push]

jobs:
  build:

    runs-on: ubuntu-latest

    strategy:
      matrix:
        node-version: [10.x]

    steps:
    - uses: actions/checkout@v1
    - name: Use Node.js ${{ matrix.node-version }}
      uses: actions/setup-node@v1
      with:
        node-version: ${{ matrix.node-version }}
    - name: npm install, lint, and test
      run: |
        npm ci
        npm run lint
        npm run test:coverage
      env:
        CI: true
```

**Comment by mahowa** [comment]
@all-contributors please add @augusto-jm-amaral to code

**Pull request merged**